crates/biome_css_semantic/src/events.rs

```diff
@@ -65,7 +65,8 @@ impl SemanticEventExtractor {
             // allowing for proper scoping and inheritance of styles.
             kind if kind == CSS_QUALIFIED_RULE
                 || kind == CSS_NESTED_QUALIFIED_RULE
-                || kind == CSS_MEDIA_AT_RULE =>
+                || kind == CSS_MEDIA_AT_RULE
+                || kind == CSS_SUPPORTS_AT_RULE =>
```

Comment:
fixed double thrown diagnostics adding this.

ref: https://github.com/biomejs/biome/issues/4756#issuecomment-2551518226
>Plus, the rule signals both bindings, while it should signal only one. (There are two diagnostics that basically say the same thing)